README.md

```diff
@@ -8,10 +8,10 @@ FedEx Rest API documentation https://developer.fedex.com/api/en-us/get-started.h
 ## Todo
 ### Services
 - [ ] Ship API
```

Comment:
I just realized that some of these should be "checked off" now, right?

Reply:
Good point. I just checked the Create shipment checkbox in [c8bcc9f](https://github.com/WhatArmy/FedexRest/pull/5/commits/c8bcc9f337a10f1dc64441cfa3a534a97f8cc38b) This MR does not address canceling shipments or tags though.

Reply:
@rmcveigh Thank you for your contribution. I will look at the code in the morning